modules/images/fetchpriority/load.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Module Name: Fetchpriority
+ * Description: Adds `fetchpriority=high` parameter to the non lazy loaded images.
+ * Experimental: Yes
+ *
+ * @since n.e.x.t
+ * @package performance-lab
+ */
+
+/**
+ * Filter image tags in content to add fetchpriority to the image tag if lazy loaded is not set.
+ *
+ * @since n.e.x.t
+ *
+ * @param string $filtered_image The filtered image.
+ * @param string $context        The context of the image.
+ *
+ * @return string image tag
+ */
+function fetchpriority_img_tag_add( $filtered_image, $context ) {
+
+	if ( 'the_content' !== $context ) {
```

Comment:
Based on my other comment and code suggestion around the featured image:
```suggestion
	if ( 'the_content' !== $context && 'the_post_thumbnail' !== $context ) {
```